dotcom-rendering/docs/architecture/030-use-percy-for-e2e-visual-regression-testing.md

```diff
@@ -1,6 +1,8 @@
 # Use Percy/Cypress for integration-style visual regression testing
 
-## Status: Accepted
+## Status: Deprecated
+
+We no longer use Percy.
```

Comment:
it might be helpful to add some more information about _why_ we deprecated this decision